driver/flags_table.c

```diff
@@ -111,6 +111,13 @@ const struct ppm_name_value clone_flags[] = {
 	{"NAME_CHANGED", PPM_CL_NAME_CHANGED},
 	{"CLOSED", PPM_CL_CLOSED},
 	{"CLONE_NEWUSER", PPM_CL_CLONE_NEWUSER},
+	{"CLONE_CHILD_CLEARTID", PPM_CL_CLONE_CHILD_CLEARTID},
+	{"CLONE_CHILD_SETTID", PPM_CL_CLONE_CHILD_SETTID},
+	{"SIGCHLD", PPM_CL_SIGCHLD},
```

Comment:
`SIGCHLD`, does not look it is a clone flag but either that on lower bits of flags an user can put any SIG* value to represent the signal sent to the parent when the child dies.

Probably we should remove it for now?